chimney/src/main/scala/io/scalaland/chimney/PartialTransformer.scala

```diff
@@ -13,7 +13,7 @@ import scala.language.experimental.macros
   * @tparam From type of input value
   * @tparam To   type of output value
   */
-trait PartialTransformer[From, To] { self =>
+trait PartialTransformer[From, To] {
 
   def transform(src: From, failFast: Boolean): partial.Result[To]
```

Comment:
Might be good to describe these parameters, if for anything else to document how user should treat `failFast` parameter if they decide to implement this trait without any helpers. Probably with a mention that is is encouraged to use a dedicated factory method to not go through all the hassle.

Reply:
will do, when working on scaladoc